docs/cmd/kn_service_update.md

```diff
@@ -68,6 +68,8 @@ kn service update NAME
       --no-lock-to-digest                 Do not keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision)
       --no-wait                           Do not wait for 'service update' operation to be completed.
   -p, --port string                       The port where application listens on, in the format 'NAME:PORT', where 'NAME' is optional. Examples: '--port h2c:8080' , '--port 8080'.
+      --probe-liveness string             Add liveness probe to Service deployment. Supported probe types are HTTGet, Exec and TCPSocket. Format: [http,https]:host:port:path;<common_opts>, exec:cmd,cmd,...;<common_opts>, tcp:host:port;<common_opts>. Common opts (comma separated, case insensitive): InitialDelaySeconds=<int_value>, FailureThreshold=<int_value>, SuccessThreshold=<int_value>, PeriodSeconds=<int_value>, TimeoutSeconds=<int_value>
```

Comment:
one general concern: Liveness and readiness probes are part of the ContainerSpec, so in case you have multiple containers you can also have multiple probes. I think we should keep it like here for the main container but also add it to the `kn container` commands (that's also one reason why we cant use pipes for the probe generation).

Reply:
That's auto-magicly parth of `kn container add` flags because logic+flags are shared from our `PodSpec` implementation.

Reply:
However, it might be a bit tricky with sidecars. 
https://github.com/knative/serving/blob/main/pkg/apis/serving/k8s_validation.go#L410-L411

Reply:
Ah, I see. So `kn container add` should *not* allow those options. Are there any other container related options that might be applied only on the main-container ? If so, we might consider to restrict the possible options on `container add` in the future. For now, I'm fine that we still have it, I guess it will lead to an error on the server side if somebody would use a probe on an extra container.